<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refresh">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nbPlayers">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="flop1_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="flop1_col">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="h1_val">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="h1_col">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="h2_val">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="h2_col">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="flop2_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="flop2_col">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="flop3_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="flop3_col">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="turn_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="turn_col">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="river_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="river_col">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♦</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>♣</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="victoire">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>381</width>
      <height>51</height>
     </size>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Victory probabilty: 0%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>My cards:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Flop:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>River:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>h1_val</tabstop>
  <tabstop>h1_col</tabstop>
  <tabstop>h2_val</tabstop>
  <tabstop>h2_col</tabstop>
  <tabstop>flop1_val</tabstop>
  <tabstop>flop1_col</tabstop>
  <tabstop>flop2_val</tabstop>
  <tabstop>flop2_col</tabstop>
  <tabstop>flop3_val</tabstop>
  <tabstop>flop3_col</tabstop>
  <tabstop>turn_val</tabstop>
  <tabstop>turn_col</tabstop>
  <tabstop>river_val</tabstop>
  <tabstop>river_col</tabstop>
  <tabstop>nbPlayers</tabstop>
  <tabstop>refresh</tabstop>
  <tabstop>victoire</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
